src/locale/locales/it/messages.po

```diff
@@ -757,11 +757,11 @@ msgstr "Aspetto"
 
 #: src/view/screens/Settings/index.tsx:475
 msgid "Appearance settings"
-msgstr ""
+msgstr "Aspetto"
 
 #: src/Navigation.tsx:326
 msgid "Appearance Settings"
-msgstr ""
+msgstr "Aspetto"
```

Comment:
```suggestion
msgstr "Impostazioni dell'aspetto"
```
Likewise, this is the title of the screen, so I think the full title is probably better?